pkg/apis/autoscaling/v1alpha1/pa_validation.go

```diff
@@ -21,13 +21,11 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/equality"
 	"knative.dev/pkg/apis"
-	"knative.dev/serving/pkg/apis/autoscaling"
 	"knative.dev/serving/pkg/apis/serving"
 )
 
 func (pa *PodAutoscaler) Validate(ctx context.Context) *apis.FieldError {
 	errs := serving.ValidateObjectMetadata(pa.GetObjectMeta()).ViaField("metadata")
-	errs = errs.Also(pa.validateMetric())
```

Comment:
Not sure if we might still want to keep this around, for backward compatibility for existing services that have already been created before this and had invalid metric annotation.

Reply:
Existing services means?
Even if service created that service won't be created successfully